### PR TITLE
Registration page Layout

### LINKED ISSUE #1: Make a registration page
### The registration page should ask for the following details:
- Personal Details
  - Name
  - Date of birth
  - Father's name
  - Mother's name
  - Guardian's name
- Contact Info
  - Email
  - Phone number
  - Address etc
- Academic Info
  - Appropriate fields to be added here
- Course info
  - Selected course details (options form a dropdown list)

#### The theme of the page should look exactly like the other pages.

### PR DESCRIPTION
**Pull request by priyank23**
This PR is in reference to issue #1 .
This adds a new page for registration of a student. It also links the page to the "Enroll Now" button in the home page.
Minor smelling errors in the about-us section is also corrected.

### CODE REVIEW

File: about_us-partial.html [hunk 1]
```diff
@@ -46,8 +46,8 @@
 
 
 <dl>
-    <dt>Vison</dt>
-    <dd>Piradius is the result oriented and pioneer
+    <dt>Vision</dt>
```
Comment (287mdsahil) [inline about_us-partial.html:49]: Since this was not related to this issue, this should have been another commit.

File: registration.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
+<style>
```
Comment (287mdsahil) [inline registration.html]: The registration page will not have the navigation bar. Also, it should open in another tab.

File: about_us-partial.html (hunk 1 above)
Comment (287mdsahil) [reply about_us-partial.html:49]: No need to change this one, 
Just mark it as resolved after you read it. :)

File: registration.html (hunk 2 above)
Comment (287mdsahil) [reply registration.html:1]: Also, rename this file as `registration-partial.html`

**Review by 287mdsahil — CHANGES_REQUESTED** [review]
rename the `registration.html` file as `registration-partial.html`

**Review by 287mdsahil — CHANGES_REQUESTED** [review]
The registration page should not have the navigation bar

**Review by 287mdsahil — CHANGES_REQUESTED** [review]
The registration page should open in a new tab

**Comment by priyank23** [comment]
@287mdsahil Check

**Comment by 287mdsahil** [comment]
LGTM

**Review by 287mdsahil — APPROVED** [review]

**Pull request merged**